catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: "Component"
+metadata:
+  name: 'event-routing-backends'
+  description: "A package for Open edX events tracking"
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: ""
+spec:
+
+  # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
+  owner: "group:edly"
```

Comment:
@salman2013 since this github group doesn't exist, can you just mae this `user:ziafazal` and we can update it to a team later if we setup a github team. Sorry I missed this the last round.